Run7/step01/flats.cfg

```diff
@@ -0,0 +1,66 @@
+[ACQUIRE]
+flat
+
+[FLAT]
+SHUTTER = OPEN  # Shutter opens at start of acqusition, closes at end
+BCOUNT=1        # number of bias frames per flat image
+WL    =red    # wavelength filter to use for the flats
+HILIM =15.0    # maximum seconds for a flat field exposure
+LOLIM =0        # minimum seconds for a flat field exposure
+FILTERCONFIG = /gpfs/slac/lsst/fs2/u1/devel/ccs/bot-eotest-configs/Run6/calib.cfg
```

Comment:
Please update this path.  The location would presumably be where this package is installed. It would be good to document that info somewhere more obvious, e.g., at the shifter instruction page or similar.